railties/lib/rails/railtie/forker.rb

```diff
@@ -0,0 +1,49 @@
+module Rails
+  class Forker
+    autoload :Runner, 'rails/railtie/forker/runner'
+
+
+    def runner=(runner)
+      @runner=runner 
+    end
+
+    def options
+      @options ||= {}
+    end
+
+    def options=(o)
+      options.merge!(o)
+    end
+
+    def before_fork(&blk)
```

Comment:
After consideration, I don't think I can apply ActiveSupport::Callbacks in this example, as it breaks the fork flow. Callbacks in this case are not designed to act around the fork, rather before the new process is forked **in master** and after the process is forked **in child**. Don't think those AS callbacks are adaptable.